lib/travis/build/script/scala.rb

```diff
@@ -6,8 +6,8 @@ class Script
       class Scala < Jvm
 
         DEFAULTS = {
-          scala: '2.10.4',
-          jdk:   'default'
+          scala: '2.12.2',
+          jdk:   'oraclejdk8'
```

Comment:
Makes sense! That'd be when trusty images become the default, right? (https://blog.travis-ci.com/2015-10-14-opening-up-ubuntu-trusty-beta/)